### PR TITLE
Support version command

### PR DESCRIPTION
**Pull request by vjerci**
## Description
This pr adds support for version command as well as hardcoding tag version in binaries.

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.

### CODE REVIEW

File: goreleaser/linux.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ builds:
     goarch:
       - 386
       - amd64
+    ldflags:
```
Comment (cfabianski) [inline goreleaser/linux.yaml:16]: Could you explain what this does please?
Just checking that the version would be the version defined in the tag and/or the commit SHA when we release

File: pkg/commands/app.go [hunk 2]
```diff
@@ -204,41 +203,23 @@ func NewConfigCommand() *cobra.Command {
...
+func NewVersionCommand(version string) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "version [flags]",
+		Use:   "version",
 		Short: "Print the version",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			showVersion(versionFormat, cmd.Version, outputWriter)
-
+			output.DefaultLogger().Msgf("curio version: %s", version)
```
Comment (cfabianski) [inline pkg/commands/app.go]: Apparently you forgot something here :)

Comment (vjerci) [reply pkg/commands/app.go]: yeah am waiting for debug pr to be approved so that i can use it.

Comment (cfabianski) [reply pkg/commands/app.go]: wait no more, it's been approved ;)

File: goreleaser/linux.yaml (hunk 1 above)
Comment (vjerci) [reply goreleaser/linux.yaml:16]: this sets golang variable  called `Version` in package `github.com/bearer/curio/cmd/curio` to a github tag.
So once it is built it should have the correct `Version` embedded inside the program.

Comment (cfabianski) [reply goreleaser/linux.yaml:16]: Could we have both a SHA **and** a Tag when available? 🙏

File: pkg/commands/app.go (hunk 2 above)
Comment (vjerci) [reply pkg/commands/app.go]: sry was paring with with elizabeth for some great code features  :)

File: goreleaser/linux.yaml (hunk 1 above)
Comment (vjerci) [reply goreleaser/linux.yaml:16]: sure

Comment (vjerci) [reply goreleaser/linux.yaml:16]: should the version command output it
```
curio version: {tag}
sha: {SHA}
```

Comment (cfabianski) [reply goreleaser/linux.yaml:16]: Yes that's perfect IMO cc @gotbadger @markmichon

File: pkg/commands/app.go [hunk 3]
```diff
@@ -215,41 +214,23 @@ func NewConfigCommand() *cobra.Command {
...
 			return nil
 		},
-		SilenceErrors: true,
-		SilenceUsage:  true,
+		SilenceErrors: false,
+		SilenceUsage:  false,
```
Comment (cfabianski) [inline pkg/commands/app.go]: Is that intended?

Comment (cfabianski) [reply pkg/commands/app.go]: https://github.com/spf13/cobra/blob/860791844ed3a2e544a9b9bbbcb14144a948ad20/command.go#L222-L226

Comment (vjerci) [reply pkg/commands/app.go]: nice catch, no its not needed its the default

File: goreleaser/linux.yaml (hunk 1 above)
Comment (vjerci) [reply goreleaser/linux.yaml:16]: implemented and resolving this one for now, if its a problematic for some reason we can always fix it later.

**Review by cfabianski — APPROVED** [review]

**Pull request merged**